### ISSUE TITLE
PHI-base FASTA file: brackets in gene names don’t have gene names, just the brackets text

### ISSUE THREAD

**Issue by martin2urban** [PHI-base/curation#1]
Communicated by Rob:
Example
I1RQN9#PHI:1189#(Gpmk1)#5518#Fusarium_graminearum#reduced_virulence
Should be MAP1 in header too.

**Comment by CuzickA** [comment]
Dear Both,

I have made a note on the spreadsheet for PHI:1211 (Sc Ctk1) that Uniprot id I1RRQ8 is obsolete, however as there is currently no replacement the data remains as it is more useful than an empty cell.

Both UniProt ids Q8X1A2 and I1RQN9 are valid for Map1. Q8X1A2 found in UniProt from the author documented Genbank ID AAL73403 and I1RQN9 from FGSG_06385. We generally try to stay true to the authors information. This can cause difficulties over time as IDs change. After discussion with Martin we decided that as both the UniProts are for the same gene/protein in the same strain PH-1 we will update the spreadsheet entries to I1RQN9 (and I’ve noted this on the spreadsheet).

Thanks for the info. It is always good to have feedback on the PHI-base data.

Alayne
